classes/02class/exercises/c02-network06/afaryy/README.md

```diff
@@ -0,0 +1,91 @@
+# c02-network06
+
+
+- Create an Elastic IP:
+```bash
+export AWS_PROFILE=devopsacademyau
```

Comment:
Nice to see you are using a profile for that :-)

Reply:
Thanks